introduction.rst

```diff
@@ -0,0 +1,12 @@
+Introduction
+================
+
+rovercode is an open-source educational robotics platform. It consists of a hardware platform (the rover's chassis and custom PCB) 
```

Comment:
Would you want to link the `custom PCB` to our Upverter? https://upverter.com/rovercode/
And maybe the chassis to the repository of the CAD files?